popup.js

```diff
@@ -1,67 +1,180 @@
-var storage_id = "storage_id"
+var skillsId = "skillsId";
+var allSkills = [];
 
-
-function send(added_exp, callback){
-  /* (int) -> (int)
-   *
-   * Gives amount of exp by which overall exp increases and returns overall exp.
+function updateExp(addedExp, callback, skillNr){
+  /* Increases skill's exp by a certain amount.
    */
-
+  var skillName = allSkills[skillNr];
```

Comment:
What if the skilNr is not in the allSkills table? How can you provide that would never happend?

Reply:
skillNr is obtained only from taking existing button's id and buttons' id are created, based on allSkills table indexes, which implies that skillNr index will have a value in allSkills table. Is it a good enough explanation? Does proving like that is a good programming practice? If so, should I insert "the proof" in a comment in the code?